apps/emqx_connector/src/emqx_connector_ssl.erl

```diff
@@ -48,3 +48,25 @@ new_ssl_config(#{<<"ssl">> := _} = Config, NewSSL) ->
     Config#{<<"ssl">> => NewSSL};
 new_ssl_config(Config, _NewSSL) ->
     Config.
+
+map_bad_ssl_error(#{pem_check := invalid_pem} = TLSLibError) ->
+    #{which_options := Paths, file_read := Reason} = TLSLibError,
+    #{
+        kind => validation_error,
+        reason => <<"bad_ssl_config">>,
+        bad_fields => Paths,
+        file_read_error => Reason
```

Comment:
Tiny nit. Maybe something like `details => emqx_utils:format("Failed to access certificate / key file: ~s", [emqx_utils:explain_posix(Reason)])` to make the error more consistent?